lib/model/smartplugin.py

```diff
@@ -21,18 +21,18 @@
 #########################################################################
 
 from lib.model.smartobject import SmartObject
-
 from lib.shtime import Shtime
 from lib.module import Modules
 import lib.shyaml as shyaml
 from lib.utils import Utils
 from lib.translation import translate as lib_translate
+
 import logging
 import os
 import threading
 import asyncio
 import time
-import inspect
+from typing import Coroutine, Any
```

Comment:
The changes in this file are much more than just "add pause_item" as the title of this PR suggests.

Reply:
Hehe... no, this is the "cleanup". @msinn added the asyncio stuff at the bottom and I reordered much of the code to make public and "need to implement"-methods more visible when reading the source code. Asyncio code was added in commit ff7f747.